### ISSUE TITLE
i have a table inside my table with the automation id "sgData" and I want to access the data inside it by clicking it. Does anyone know how to do that?

### ISSUE THREAD

**Issue by NikitaTurakhia** [microsoft/WinAppDriver#1844]

**Comment by pm90pl** [comment]
Are elements visible in the visual tree (e.g. Inspect.exe)? Hard to say if the controls were exposed to UI Automation.

**Comment by NikitaTurakhia** [comment]
@pm90pl No. They are not visible. It just shows pane for the whole table.

**Comment by varun210619** [comment]
@NikitaTurakhia Can you please show(screenshot if possible) what you get in inspect.exe.
On which technology your application is based?

Some workarounds - 

1. If you cannot get any row, can you get any option in application to export data and then work?
2. You can also try keyboard actions on application - pressing tabs and then clicks on particular rows.
